### PR TITLE
oded/act-580

### PR DESCRIPTION
**Pull request by odeds**

### CODE REVIEW

File: src/components/znkExercise/core/directive/znkSwiperDrv.js [hunk 1]
```diff
@@ -81,9 +81,12 @@
                         });
                     }
 
-                    function postLink(scope,element){
+                    function postLink(scope,element,attrs,ngModelCtrl){
```
Comment (assafshp) [inline src/components/znkExercise/core/directive/znkSwiperDrv.js]: as far as I see no one uses attrs... where is eslint ??

File: src/components/znkExercise/core/directive/znkSwiperDrv.js [hunk 2]
```diff
@@ -81,9 +81,12 @@
...
                         $timeout(function(){
-                            defer.resolve(new Swiper(element[0]));
+                            var currSlideIndex = ngModelCtrl.$viewValue;
+                            defer.resolve(new Swiper(element[0], {
+                                initialSlide: currSlideIndex || 0
```
Comment (assafshp) [inline src/components/znkExercise/core/directive/znkSwiperDrv.js]: what does it mean for swiper to start with slide 0?

**Comment by assafshp** [comment]
conflict

File: src/components/znkExercise/core/directive/znkSwiperDrv.js (hunk 1 above)
Comment (odeds) [inline src/components/znkExercise/core/directive/znkSwiperDrv.js]: ye but im using the ngModelCtrl and it's not inject services

File: src/components/znkExercise/core/directive/znkSwiperDrv.js (hunk 2 above)
Comment (odeds) [inline src/components/znkExercise/core/directive/znkSwiperDrv.js]: starts at first slide. it's just a precaution cause always should be currSlideIndex

**Pull request merged**